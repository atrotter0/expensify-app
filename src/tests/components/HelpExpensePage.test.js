import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import HelpPage from '../../components/HelpExpensePage';

test('should render HelpPage', () => {
  const wrapper = shallow(<HelpPage />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
