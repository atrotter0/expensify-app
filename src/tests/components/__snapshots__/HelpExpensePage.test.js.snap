// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render HelpPage 1`] = `
<div>
  <p>
    This is the help component
  </p>
</div>
`;
